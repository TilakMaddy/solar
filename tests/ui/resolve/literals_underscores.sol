contract LT {
    uint256 a = 1000_; //~ERROR: invalid use of underscores in number literal
    uint256 b = 100__0;//~ERROR: invalid use of underscores in number literal
    uint256 c = 1_.4e10;//~ERROR: invalid use of underscores in number literal
    uint256 d = 3.4_e10;//~ERROR: invalid use of underscores in number literal
    uint256 e = 3.4e_10;//~ERROR: invalid use of underscores in number literal

    // exception
    uint256 f = 3._4e10; // Does not show up
}
