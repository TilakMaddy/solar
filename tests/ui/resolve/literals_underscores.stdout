"1000_"
"100__0"
"1_.4e10"
"3.4_e10"
"3.4e_10"
